adyen.gemspec

```diff
@@ -39,5 +38,5 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files = ['README.md', 'CHANGELOG.md', 'CONTRIBUTING.md']
 
   s.files = `git ls-files`.split($/)
-  s.test_files = s.files.grep(%r{^(test|spec|features)/})
```

Comment:
This is the standard way bundler generates the gemspec file which I just left as it was generated. If you're going to remove `spec`, you can remove `features` as well